package/CHANGELOG

```diff
@@ -13,12 +13,14 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/16 jandom
+??/??/16 jandom, richardjgowers
 
-  * 0.14.1
+  * 0.15.0
 
 Fixes
 
+ * Fixed Atom.position/velocity/force returning a view onto Timestep array
```

Comment:
Well we return a copy now instead of a view. So just say in the CHANGELOG that we return a copy now for this.